### ISSUE TITLE
Get text divided into paragraphs?

### ISSUE THREAD

**Issue by sindresorhus** [ggml-org/whisper.cpp#539]
This would be useful when transcribing to a text document because having the text divided into paragraphs makes it more readable. This may be outside the scope of this project. Just thought I would ask.

**Comment by ggerganov** [comment]
Yeah, this sounds like out of scope. Probably there is some 3rd party tool that you can apply on the output of whisper.cpp

**Comment by abodacs** [comment]
@sindresorhus you can try build in javascript segmenter
https://www.stefanjudis.com/today-i-learned/how-to-split-javascript-strings-with-intl-segmenter/

**Comment by sindresorhus** [comment]
@abodacs The linked segmenter only covers sentences, not paragraphs. I'm already doing sentence segmentation in my app.

**Comment by abodacs** [comment]
@sindresorhus Aha,
you can check this discussion 
https://github.com/openai/whisper/discussions/552
